packages/devops/scripts/ci/deployFrontend.sh

```diff
@@ -4,7 +4,7 @@ DIR=$(dirname "$0")
 DEPLOYMENT_URL=$(${DIR}/determineDeploymentUrl.sh)
 if curl --output /dev/null --silent --head --fail $DEPLOYMENT_URL; then
     echo "Deployment for ${CI_BRANCH} exists, updating with latest changes"
-    ssh ubuntu@$DEPLOYMENT_URL "cd tupaia; yarn workspace @tupaia/${PACKAGE} build"
+    ssh -o ServerAliveInterval=15 ubuntu@$DEPLOYMENT_URL "cd tupaia; yarn workspace @tupaia/${PACKAGE} build"
```

Comment:
Sends a null packet to the server every 15 seconds if no data is received, which should help to keep the ssh connection alive and receiving logs

Reply:
Note that the default value of `ServerAliveCountMax` is 3 so the timeout on an actual connection problem will now be 45 seconds instead of whatever the default is.

Reply:
Yep good point, but happy with that